supplementary/devices/index.rst

```diff
@@ -26,12 +26,13 @@ All of these should only be document in the exact same way here and **only an ov
    T00F
    titan
    vegeta
+   wt88047
    yuga
    Z00L
    
 
-How to document
----------------
+How to document 
```

Comment:
Plesae, don't add trailing spaces.

Also, the length of the line should be same as the length of the text

Reply:
Agreed, don't change things without good reason.

Reply:
@UniversalSuperBox can there be a good reason to add trailing spaces lol?